.travis.yml

```diff
@@ -79,9 +79,25 @@ matrix:
           env:
             - PX4_DOCKER_REPO=px4io/px4-dev-simulation-bionic:2019-10-04
             - BUILD="source ./scripts/validate_sdf.bash"
-        - name: MacOSX 10.14 (Xcode 11.0) build with Gazebo 9
+        - name: MacOSX Sierra (Xcode 9.2) build with Gazebo 7
```

Comment:
I would drop this version. Supporting 2 and with Catalina 3 is enough.

Reply:
I am adding this one so we can have a build pipe for Gazebo 7 also in MacOSX.